### PR TITLE
Add copy button for copying guide code examples [skip ci]

### PR DESCRIPTION
**Pull request by p8**
### Summary

Instead of carefully selecting the code in examples to copy it, use a copy button
for copying the code. This uses https://clipboardjs.com/ for copying.

For the bash code examples only lines starting with $ prompt are copied.
The $ prompt itself is not copied.
So for the following only `ruby --version` is copied.

image

If the copy was successful the button text changes to "Copied!" for 3 seconds.

### CODE REVIEW

**Comment by simi** [comment]
Would it be possible to also show some notification it was successfully copied? I think usually this is done using tooltip.

**Comment by p8** [comment]
Yes, was going to add that before making this PR ready for review...

**Comment by simi** [comment]
Alternatively it could be easier to change text to `Copied` for few seconds.

**Comment by p8** [comment]
Yes, I think that would be the easier solution.

File: guides/assets/stylesheets/main.css [hunk 1]
```diff
@@ -238,6 +238,26 @@ body {
...
+
+.clipboard-button:active {
+  background: #ccc;
+}
+
+.clipboard-content {
+  opacity:0;
+  position:absolute;
+  z-index:-1;
+}
```
Comment (eugeneius) [inline guides/assets/stylesheets/main.css:259]: Looks like this is missing from `main.rtl.css`.

Comment (p8) [reply guides/assets/stylesheets/main.css:259]: Missed that. Thanks! 😄

**Comment by eugeneius** [comment]
Thanks, this is great!

**Pull request merged**